distribute/distribute.sh

```diff
@@ -51,6 +51,18 @@ then
 fi
 
 
+echo "****************************************************************"
```

Comment:
this whole script is meant for packaging some `deb` and the like. 
how do you package this for FreeBSD? and how would it's content differ from a Linux package?
I'd prefer to handle this in another PR though...

Reply:
Packaging is not needed at this point I think. The reason I wanted to use your logic their was because I need to do this somewhere:

`sed -i -e 's@"/lib/udev/@"/usr/local/share/inkscape/extensions@' files/silhouette-udev.rules`

and also override UDEV and PREFIX in the makefile to match FreeBSD.

`make install UDEV=/usr/local/lib/udev PREFIX=/usr/local`

Any suggestion where to put this ?

Reply:
does `notify-send` indeed work and can it be triggered through `udev` on FreeBSD?
FYI: I don't use it neither on my system.